provider/datagen/src/lib.rs

```diff
@@ -69,10 +70,23 @@ mod registry;
 mod source;
 mod transform;
 
+use elsa::sync::FrozenMap;
 pub use error::{is_missing_cldr_error, is_missing_icuexport_error};
+use icu_locid::LanguageIdentifier;
+use icu_locid_transform::fallback::LocaleFallbackConfig;
+use icu_locid_transform::fallback::LocaleFallbacker;
+use icu_provider::datagen::*;
+use icu_provider::prelude::*;
+use memchr::memmem;
+use options::{FallbackMode, LocaleInclude};
 #[allow(deprecated)] // ugh
 pub use registry::{all_keys, all_keys_with_experimental, deserialize_and_measure, key};
 pub use source::SourceData;
+use std::borrow::Cow;
```

Comment:
Nit: please keep public reexports and module-level private imports separated. We might even want to move the logic into a separate `provider.rs` file.

Reply:
They were after the prelude